app/services/personas.js

```diff
@@ -0,0 +1,17 @@
+const { listPersonas, listPersona } = require('../api/personas')
+
+const getPersonas = async (request, project = 'mcu', type) => {
+  const responses = await listPersonas(project, type)
```

Comment:
Could we rename this to `personas`

Reply:
this can be changed, but as it is, it is consistent with the naming conventions in models, knowledge and prompts services.